cmd/format.go

```diff
@@ -1,22 +1,43 @@
 package cmd
 
 import (
-	"fmt"
-
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
+
+	"github.com/elastic/elastic-package/internal/formatter"
+	"github.com/elastic/elastic-package/internal/packages"
 )
 
+const failFastFlagName = "fail-fast"
+
 func setupFormatCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "format",
 		Short: "Format the package",
 		Long:  "Use format command to format the package files.",
 		RunE:  formatCommandAction,
 	}
+	cmd.Flags().BoolP(failFastFlagName, "f", false, "fail if any file requires formatting")
 	return cmd
 }
 
 func formatCommandAction(cmd *cobra.Command, args []string) error {
-	fmt.Println("format is not implemented yet.")
+	packageRoot, ok, err := packages.FindPackageRoot()
+	if err != nil {
+		return errors.Wrap(err, "locating package root failed")
+	}
+	if !ok {
+		return errors.New("package root not found")
+	}
+
+	ff, err := cmd.Flags().GetBool(failFastFlagName)
+	if err != nil {
+		return errors.Wrapf(err, "flag not found (flag: %s)", failFastFlagName)
+	}
+
+	err = formatter.Format(packageRoot, ff)
+	if err != nil {
+		return errors.Wrapf(err, "formatting the integration failed (path: %s, failFast: %t)", packageRoot, ff)
```

Comment:
Since the user will be familiar with `--fail-fast` not `failFast`.

```suggestion
		return errors.Wrapf(err, "formatting the integration failed (path: %s, --fail-fast: %t)", packageRoot, ff)
```

Reply:
In this case I'm not convinced, it's just logging an error.